examples/magento-graphcms/pages/wishlist/index.tsx

```diff
@@ -28,6 +27,12 @@ type GetPageStaticProps = GetStaticProps<LayoutOverlayProps, Props>
 function WishlistPage() {
   const wishlistItemsData = useWishlistItems()
 
+  const isMobile = useMediaQuery<Theme>((theme) => theme.breakpoints.down('md'), {
+    defaultMatches: false,
+  })
+
+  const size: WishlistItemActionCardProps['size'] = isMobile ? 'small' : 'large'
```

Comment:
Do we really need to do this? In my opinion we need to have a size that looks good on mobile and desktop. `useMediaQuery` isn't a good practice and probably doesn't belong in the example directory